src/server/requestHandlers.js

```diff
@@ -159,8 +191,15 @@ sjrk.storyTelling.server.handleSaveStoryWithBinaries = function (request, dataSo
     }
 };
 
-// Persist the story model to couch, with the updated
-// references to where the binaries are saved
+/**
+ * Persist the story model to couch, with the updated references to where the binaries are saved
+ *
+ * @param {Object} dataSource - a Kettle CouchDB DataSource for saving stories

```

Comment:
The type should be `{Component}` rather than `{Object}`. Describe what component instance is expected too. For example, "an instance of kettle.dataSource.CouchDB". 

This comment applies thru the entire pull request. Thanks.